README.md

```diff
@@ -44,11 +44,11 @@ yarn start
 
 > Ensure you are not using Power shell
 
-6. 🎉 Open your browser and go to http://localhost:8000/ or http://localhost:8000/___graphql
+6. 🎉 Open your browser and go to http://localhost:8000/ or http://localhost:8000/\_\_\_graphql
```

Comment:
Could you rollback this line?
Because the link was corrected

Reply:
@ming-tsai I corrected that 👍🏽